pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -702,11 +702,22 @@ type ClientSpec struct {
 }
 
 type CleanupPolicySpec struct {
-	Confirmation CleanupConfirmationProperty `json:"confirmation,omitempty"`
+	Confirmation  CleanupConfirmationProperty `json:"confirmation,omitempty"`
+	SanitizeDisks SanitizeDisksSpec           `json:"sanitizeDisks"`
 }
 
 type CleanupConfirmationProperty string
 
+type SanitizeDataSourceProperty string
+
+type SanitizeMethodProperty string
+
+type SanitizeDisksSpec struct {
+	Method     SanitizeMethodProperty     `json:"method,omitempty"`
+	DataSource SanitizeDataSourceProperty `json:"dataSource,omitempty"`
+	Iteration  *int32                     `json:"iteration,omitempty"`
```

Comment:
Why use a pointer? If it's not set (0), we could just default to 1.

Reply:
That works too, no particular preference.